scripts/treedb_vdbbench_search_existing_index.py

```
@@ -0,0 +1,680 @@
+        fail("authorized search service binary checksum mismatch")
+    paths = (
+        args.query_json, args.metadata_out, args.diagnostic_response_out,
+        args.production_response_out, args.diagnostic_result, args.production_result,
+        args.diagnostic_origin_out, args.production_origin_out, args.command_ledger,
+        args.search_isolation_out,
+    )
+    for path in paths:
+        try:
+            path.resolve().relative_to(args.artifact_root)
+        except ValueError:
+            fail(f"search evidence path escapes artifact root: {path}")
```

Ruff (0.16.3)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)